deployments/helm/hephaestus/values.yaml

```diff
@@ -126,8 +121,10 @@ controller:
       pullPolicy: IfNotPresent
       pullSecrets: []
 
-    # Maximum number of concurrent builds which can be run
-    imageBuildMaxConcurrency: 5
```

Comment:
why, you would have to update the image without the chart. which we would really want to catch.
But I can add it...

Reply:
I guess I was thinking of someone who just did (wrongly, since it's a top-level)
```
release_overrides:
  hephaestus:
    chart_values:
      controller:
        imageBuildMaxConcurrency: 10
```
Maybe the better alternative is a `fail`?

Reply:
I think fail is good.

Reply:
extra fields will cause a failure.